src/includes/control-datatypes.h

```diff
@@ -0,0 +1,14 @@
+#ifndef DINIT_CONTROL_DATATYPES_H
+#define DINIT_CONTROL_DATATYPES_H
+
+#include <cstdint>
+
+namespace dinit_ctypes {
```

Comment:
`ctypes` sounds a bit too close to the existing character types, maybe `cptypes` or `cp_types` (for "command protocol types")?

Reply:
seems reasonable to avoid confusion. it's dinit_cptypes now